src/Teams/Views/MultipleAnswerQuestion/EditMultipleAnswerQuestion.cshtml

```diff
@@ -0,0 +1,195 @@
+            buttonForRemove.setAttribute('onclick', 'RemoveAnswer(this)');
+            buttonForRemove.setAttribute('id', idCount);
+            div.setAttribute('id', 'div' + idCount);
+
+            element.appendChild(div);
+            div.appendChild(checkbox);
+            div.appendChild(inpt);
+            div.appendChild(buttonForRemove);
+            div.appendChild(br);
+            div.appendChild(br);
+
+            answersCount++;
+            idCount++;
+           
+            if (answersCount == MaxAnswersNumber) {
+                buttonForAdd.hidden = true;
+            }
+        }
+
+
+    }
+
+
+    function addAnswer() {
+        let element = document.getElementById('options_container');
+        let inpt = document.createElement('input');
+        let checkbox = document.createElement('input');
+        let br = document.createElement('br');
+        let buttonForAdd = document.getElementById('buttonForAdd');
+        let buttonForRemove = document.createElement('input');
+        let div = document.createElement('div');
+
+        inpt.setAttribute('type', 'text');
+        inpt.setAttribute('class', 'answersText');
+        checkbox.setAttribute('type', 'checkbox');
+        checkbox.setAttribute('class', 'itTrue');
+        buttonForRemove.setAttribute('type', 'button');
+        buttonForRemove.setAttribute('value', 'X');
+        buttonForRemove.setAttribute('class', 'buttonForRemove btn btn-danger btn-sm');
+        buttonForRemove.setAttribute('onclick', 'RemoveAnswer(this)');
+        buttonForRemove.setAttribute('id', idCount);
+        div.setAttribute('id', 'div' + idCount);
+
+        element.appendChild(div);
+        div.appendChild(checkbox);
+        div.appendChild(inpt);
+        div.appendChild(buttonForRemove);
+        div.appendChild(br);
+        div.appendChild(br);
+
+
+        answersCount++;
+        idCount++;
+        if (answersCount == MaxAnswersNumber ){
+            buttonForAdd.hidden = true;
+        }
+
+    }
+
+    function RemoveAnswer(obj) {
+        let buttonForAdd = document.getElementById('buttonForAdd');
+        let divForRemove = document.getElementById('div' + obj.id);
+        divForRemove.remove();
+        buttonForAdd.hidden = false;
+        answersCount--;
+    }
+
+    function addQuestion() {
+        let someText = document.getElementsByName('questionText');
+        let questionText = someText[0].value;
+        let answersTextList = document.getElementsByClassName('answersText');
+        let answersCheckboxList = document.getElementsByClassName('itTrue');
+        var textMassive = new Array();
+        var checkboxValueMassive = new Array();
+        let isChecked = 0;
+
+        for (let i = 0; i < answersTextList.length; i++) {
+            textMassive[i] = answersTextList[i].value;
+            if (answersCheckboxList[i].checked) {
+                checkboxValueMassive[i] = true;
```

Comment:
What do you think about this assignment checkboxValueMassive[i] = answersCheckboxList[i].checked instead of assigning true/false in different "else" branches?